restful-tango/tangoREST.py

```diff
@@ -137,9 +137,11 @@ def convertJobObj(self, dirName, jobObj):
 		# List of input files
 		input = []
 		for file in jobObj['files']:
+            inFile = file['localFile']
+            vmFile = file['destFile']
```

Comment:
It looks like we're mixing tabs and spaces. Considering this is Python, that should probably be fixed.